lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -160,68 +160,105 @@
 '''
 
 EXAMPLES = '''
-  - name: Create a new provider in ManageIQ ('Hawkular' metrics)
-    manageiq_provider:
-      name: 'EngLab'
-      type: 'OpenShift'
-      provider:
-        auth_key: 'topSecret'
-        hostname: 'example.com'
-        port: 8443
-        verify_ssl: False
-      metrics:
-        role: 'hawkular'
-        hostname: 'example.com'
-        port: 443
-        verify_ssl: False
-      manageiq_connection:
-        url: 'http://127.0.0.1:3000'
-        username: 'admin'
-        password: 'smartvm'
-        verify_ssl: False
-
-  - name: Update an existing provider named 'EngLab' (defaults to 'Prometheus' metrics)
-    manageiq_provider:
-      name: 'EngLab'
-      type: 'Openshift'
-      provider:
-        auth_key: 'verySecret'
-        hostname: 'next.example.com'
-        port: 8443
-        verify_ssl: False
-      metrics:
-        hostname: 'next.example.com'
-        port: 443
-        verify_ssl: False
-      manageiq_connection:
-        url: 'http://127.0.0.1:3000'
-        username: 'admin'
-        password: 'smartvm'
-        verify_ssl: False
-
-  - name: Delete a provider in ManageIQ
-    manageiq_provider:
-      state: 'absent'
-      name: 'EngLab'
-      manageiq_connection:
-        url: 'http://127.0.0.1:3000'
-        username: 'admin'
-        password: 'smartvm'
-        verify_ssl: False
-
-  - name: Create a new Amazon provider in ManageIQ using token authentication
-    manageiq_provider:
-      name: 'EngAmazon'
-      type: 'Amazon'
-      provider_region: 'us-east-1'
-      provider:
-        hostname: 'amazon.example.com'
-        userid: 'hello'
-        password: 'world'
-      manageiq_connection:
-        url: 'http://127.0.0.1:3000'
-        token: 'VeryLongToken'
-        verify_ssl: False
+  - hosts: 127.0.0.1
+    tasks:
+    - name: Create a new provider in ManageIQ ('Hawkular' metrics)
+      manageiq_provider:
+        name: 'EngLab'
+        type: 'OpenShift'
+        state: 'present'
+        provider:
+          auth_key: 'topSecret'
+          hostname: 'example.com'
+          port: 8443
+          verify_ssl: false
+          security_protocol: 'ssl-with-validation-custom-ca'
+          certificate_authority: |
+          -----BEGIN CERTIFICATE-----
+          "certificate goes here"
+          -----END CERTIFICATE-----
+        metrics:
+          auth_key: 'topSecret'
+          role: 'hawkular'
+          hostname: 'example.com'
+          port: 443
+          verify_ssl: false
```

Comment:
our mistake :-) it's verfy_ssl: true ... because we have  'ssl-with-validation-custom-ca' in the next line ...

I asked @ilackarms to fix this in his playbook too ...